frontend/components/BlogPreview.js

```diff
@@ -0,0 +1,91 @@
+//All necessary imports for this javascript
+import React from "react";
+import styles from "../styles/Blog.module.css";
+import Link from "next/link";
+import Chip from "@material-ui/core/Chip";
+import { useStyles } from "../data/BlogData";
+
+const BlogPreview = ({ individualBlogPreview }) => {
+  const classes = useStyles();
+
+  const execs = [
+    "/portraits/blog-authors/vivianw2021.jpg",
+    "/portraits/blog-authors/georgie2021.jpg",
+  ];
+  const { date, img, heading, subheading, blog_no, category } =
+    individualBlogPreview.fields;
+
+  const authors = individualBlogPreview.authors;
+
+  const imgUrl = "https:" + img.fields.file.url;
```

Comment:
is this intended behaviour 

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/80564580/201577511-c39141bc-aec6-40c7-aec0-c5d3659ce867.png">

happens whenever i navigate to the 4th page of the pagination component